devtools/create_achievements/steam_achievements.py

```diff
@@ -66,7 +71,14 @@
 parser.add_argument("--steamid", required=True, default="631570", type=int, help="Steam game id")
 parser.add_argument("--saveasgalaxyid", type=int, help="GOG Galaxy game id")
 parser.add_argument("-v", "--verbose", action="store_true")
+parser.add_argument("-d", "--debug", action="store_true")
 args = parser.parse_args()
+service = Service("/usr/bin/chromedriver")
```

Comment:
According to the [documentation](https://chromedriver.chromium.org/getting-started), the path is optional. If not specified, WebDriver searches the PATH for chromedriver.